### PR TITLE
Adds subscription for contacts on chat page

### PR DESCRIPTION
**Pull request by VishalZ123**
### Summary
This adds the feature of updating the contact card on the chat screen when a user sends or receives messages.

### Changes made
- Adds contact field in `MESSAGE_RECEIVED_SUBSCRIPTION` & `MESSAGE_RECEIVED_SUBSCRIPTION` subscriptions.
- Adds function to update the cache when a message is sent or received.

### Issues
Closes: #139 

@mdshamoon Please review this PR.

### CODE REVIEW

File: screens/Chat.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { useState, useEffect } from 'react';
+/* eslint-disable prettier/prettier */
```
Comment (mdshamoon) [inline screens/Chat.tsx:1]: Why are we disabling this?

File: screens/Chat.tsx [hunk 2]
```diff
@@ -8,6 +9,70 @@ import { GET_CONTACTS } from '../graphql/queries/Contact';
...
+
+  let contactId:string;
+  let id:string;
+  let body;
+  let contact;
+  switch (action) {
+    case 'RECEIVED':
+      contactId = subscriptionData.data.receivedMessage.contact.id;
+      id = subscriptionData.data.receivedMessage.id;
+      body = subscriptionData.data.receivedMessage.body;
```
Comment (mdshamoon) [inline screens/Chat.tsx:32]: I think this is similar to what we have done in message list. Can you create a common functiona and use in both places.

**Review by mdshamoon — CHANGES_REQUESTED** [review]

File: screens/Chat.tsx (hunk 1 above)
Comment (VishalZ123) [reply screens/Chat.tsx]: It was added by mistake, removed it

File: screens/Chat.tsx (hunk 2 above)
Comment (VishalZ123) [reply screens/Chat.tsx]: Implemented

**Comment by ajaman190** [comment]
@VishalZ123 can you please fix all deep scan problems first.

**Review by mdshamoon — APPROVED** [review]

**Pull request merged**